src/components/common/SummaryOfficeAddresses.vue

```diff
@@ -17,16 +17,20 @@
       <!-- Registered Mailing Address -->
       <li class="address-list-container">
         <div class="meta-container">
-          <label></label>
-          <div class="meta-container__inner"
+          <label />
```

Comment:
FYI, this (and line 55) is required for the CSS selectors at the bottom of this file.